docs/import-from-github.mdx

```diff
@@ -1,54 +1,80 @@
 ---
-title: Import from GitHub
-description: 'Import your machines from GitHub into Stately Studio.'
+title: Connect GitHub repo
```

Comment:
I may have missed a conversation about this but is this like a "Docs title" for the feature as opposed to "GitHub Sync" we use elsewhere?

Reply:
After reading the rest of the p age I see this is really the button's name so it makes more sense as a lead in. Anyway, I was just wondering because it stood out (3 words longer, bold) in the left sidebar.

Reply:
@kevinmaes it’s because it’s not named GitHub sync anywhere in the interface. This is a bit like the Stately Sky situation, where folks didn’t know where to find it because while we were calling it Stately Sky publicly, nowhere in the Studio said Stately Sky. I’m trying to avoid that problem by making the docs naming follow the UI naming!

Reply:
That sounds good!